index.html

```diff
@@ -12,10 +12,101 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
```

Comment:
For child tags it is best to indent by one tab. The head and body tags are both children of the html tag and so should be tabbed in one. I know this was in the initial code you were given so not really anything you've done wrong. Just thought I would point this out as it is best practise.